Projects.js

```diff
@@ -0,0 +1,208 @@
+const projects = [{
+  id: 1,
+  card: 'card1',
+  image: './images/SnapshootPortfolio.svg',
+  Projectname: 'Tonic',
+  history: {
+    proName: 'CANOPY',
+    tech: 'Backend Dev',
+    date: 2015,
+  },
+  disc: 'A daily selection of privately personalized reads;no accounts or sign-ups required.',
+  tags: {
+    html: 'HTML',
+    css: 'CSS',
+    javascript: 'JavaScript',
+  },
+  button: 'myBtn',
+},
+{
+  id: 2,
+  card: 'card2',
+  image: './images/multiPostStories.svg',
+  Projectname: 'Multi-Post',
+  history: {
+    proName: 'CANOPY',
+    tech: 'Backend Dev',
+    date: 2015,
+  },
+  disc: 'A daily selection of privately personalized reads;no accounts or sign-ups required.',
+  tags: {
+    html: 'HTML',
+    css: 'CSS',
+    javascript: 'JavaScript',
+  },
+  button: 'myBtn',
+},
+{
+  id: 3,
+  card: 'card3',
+  image: './images/SnapshootPortfolio.svg',
+  Projectname: 'Tonic',
+  history: {
+    proName: 'CANOPY',
+    tech: 'Backend Dev',
+    date: 2015,
+  },
+  disc: 'A daily selection of privately personalized reads;no accounts or sign-ups required.',
+  tags: {
+    html: 'HTML',
+    css: 'CSS',
+    javascript: 'JavaScript',
+  },
+  button: 'myBtn',
+},
+{
+  id: 4,
+  card: 'card4',
+  image: './images/multiPostStories.svg',
+  Projectname: 'MultiPost',
+  history: {
+    proName: 'CANOPY',
+    tech: 'Backend Dev',
+    date: 2015,
+  },
+  disc: 'A daily selection of privately personalized reads;no accounts or sign-ups required.',
+  tags: {
+    html: 'HTML',
+    css: 'CSS',
+    javascript: 'JavaScript',
+  },
+  button: 'myBtn',
+},
+];
+
+const section = document.getElementById('works');
+
+const projectMethod = () => {
+  projects.map((card) => {
+    const project = document.createElement('div');
+    project.classList.add('card_works');
+    project.classList.add(card.card);
+    project.innerHTML = `<img
+              class="card_image "
+              src=${card.image}
+              alt="Tonic Project"
+              />
+              <div class="card_detail">
+              <div class="card_title">
+              <h2>${card.Projectname}</h2>
+                <div class="card_history">
+                  <div class="card_history_detail"><p>${card.history.proName}</p></div>
+                <div>
+                   <img src="./images/Counter.svg" alt="dot" />
```

Comment:
- [ ] You have done great so far, :+1: :+1:  but there are some changes you need to make to your UI before moving to the next milestone.
- - As part of the requirement for this milestone you need to replicate the given design as much as you can, currently, in your desktop version of the popup, the technology list and the buttons are aligned to the ring according to the Figma design, but it's not same in your project, I'll suggest you update this. :+1: :smile: 

- - Your Design:
![Selection_067](https://github.com/sheezu900/My_Potfolio/assets/107632780/955e6c9d-cca9-46f7-9634-0736bd1234ce)

- - Figma design:
![Screenshot from 2023-06-21 16-07-00](https://github.com/sheezu900/My_Potfolio/assets/107632780/904d3f6d-dc15-4b2a-80ca-49914514c0f0)


- - Also according to the Figma design, in the mobile version, there is a Horizontal line between the text and the buttons as shown in the Figma design, I'll suggest you update this in your design too. :+1: :+1: 
- - Figma design:
![Selection_068](https://github.com/sheezu900/My_Potfolio/assets/107632780/2f3a7644-1541-437c-83cc-b4839f8aaa51)